### DISCUSSION TITLE
Upgrading Cake.Yarn to Cake 2.0

### DISCUSSION THREAD

**Discussion by paulmilbank** [cake-build/cake#3888] [Extension Q&A] [answered]
Is it possible to get some eyes on https://github.com/cake-contrib/cake-yarn/pull/42 and the issue https://github.com/cake-contrib/cake-yarn/issues/44 please? There has been no response to my or another PR on the repo. The tests all pass after the 2.0 upgrade on my machine

**Comment by augustoproiete** [comment]
pinging @MilovanovM who's the maintainer of Cake.Yarn

**Comment by augustoproiete** [comment, marked as answer]
Thanks for the PR @paulmilbank. Your PR was merged a few hours ago and released with Cake.Yarn [v1.0.0](https://github.com/cake-contrib/Cake.Yarn/releases/tag/1.0.0) which targets Cake v2.0.0.

Also, a new release of Cake.Yarn ([v2.0.0](https://github.com/cake-contrib/Cake.Yarn/releases/tag/1.0.0)) shipped right after, and it upgrades the addin for compatibility with Cake v3.0.0

- :rocket: [GitHub Releases](https://github.com/cake-contrib/Cake.Yarn/releases)
- :package: [NuGet Packages](https://www.nuget.org/packages/Cake.Yarn)
